repository/Seaside-WebSocket-Core.package/WAWebSocketJSPusher.class/instance/websocketEventHandlerJavaScriptOn..st

```diff
@@ -0,0 +1,4 @@
+protected
+websocketEventHandlerJavaScriptOn: aStream
+	aStream 
+		nextPutAll: '.addEventListener(''message'', (event) => { Function(`"use strict";return (${event.data})`)(); })'
```

Comment:
why the need for a function?

Reply:
Because 'eval' is bad :-). https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/eval#never_use_eval!